src/hashdict.jl

```diff
@@ -254,6 +254,19 @@ function _compact_order{K,V}(h::HashDict{K,V,Ordered})
     nothing
 end
 
+"""
+```
+sizehint!(d, n)
```

Comment:
Is it necessary to re-define doc strings for functions like this, where there is already a definition in base Julia? Also, the argument names don't match up.